app/controllers/api/v1/api_controller.rb

```diff
@@ -1,15 +1,15 @@
 module Api
   module V1
-    class ApiController < ApplicationController
-      include Api::Concerns::ActAsApiRequest
```

Comment:
is this concern used somewhere else?

Reply:
Yes in Devise controllers used from the API.

>One consideration: Devise controllers will always
inherit from ApplicationController, hence we still need to use the
ActAsApiRequest concern in Devise controllers. This is because Devise is shared
between the API-only app and the admin panel.